ucum-java8/src/main/java/systems/uom/ucum/format/UCUMFormat.java

```diff
@@ -207,24 +207,48 @@ public Appendable format(Unit<?> unknownUnit, Appendable appendable)
 		    temp.append(')');
 		}
 		int pow = productUnits.get(u);
-		if (app.length() > 0) { // Not the first unit.
-		    if (pow >= 0) {
-			app.append('.');
-		    } else {
-			app.append('/');
-			pow = -pow;
-		    }
+				int indexToAppend;
+				if(app.length() > 0) { // Not the first unit.
+
+					if(pow >= 0) {
+
+						if(app.indexOf("1/") >= 0) {
+							indexToAppend = app.indexOf("1/");
+							app.replace(app.indexOf("1/"), app.indexOf("1/") + 2, "/");
```

Comment:
My point is that you already have the value of `app.indexOf("/")` in `indexToAppend` in the line above. So can't you just say `app.replace(indexToAppend, indexToAppend+2, "/")`? Why do you need to calculate the same value two more times?